Apps/Database/src/Delegates/DBNoteDelegate.cs

```diff
@@ -47,13 +47,13 @@ public class DBNoteDelegate : INoteDelegate
         }
 
         /// <inheritdoc />
-        public DBResult<Note> GetNote(Guid noteId)
+        public DBResult<Note> GetNote(Guid noteId, string hdid)
```

Comment:
Is the hdid necessary?

Reply:
We made the notes table a composite key with ID + HDID, so we do need it!